docker-compose.yml

```diff
@@ -1,4 +1,22 @@
 name: ${COMPOSE_PROJECT_NAME:-cht-sync}
+version: '3.8'
```

Comment:
just a quickie one off  because you haven't requested a formal review yet! This change avoids the startup `WARN`:

> WARN[0000]  . /cht-sync/docker-compose.yml: the attribute `version` is obsolete, it will be ignored, please remove it to avoid potential confusion 



```suggestion
```